x-pack/test/functional/apps/transform/creation_index_pattern.ts

```diff
@@ -277,7 +328,7 @@ export default function ({ getService }: FtrProviderContext) {
         });
 
         it('navigates through the wizard and sets all needed fields', async () => {
-          await transform.testExecution.logTestStep('displays the define pivot step');
+          await transform.testExecution.logTestStep('displays the define step');
```

Comment:
I think we should call `assertSelectedTransformFunction` to validate that the default transform function selection is `pivot`, maybe even with a separate test step message.
Picking this code line as one example - would be good to validate the selected transform function whenever we open the wizard, also on clone.

Reply:
Added in https://github.com/elastic/kibana/pull/86966/commits/d8c8fe2553be93b2b01bee2da8f1f4c77f4142f4